### PR TITLE
Add sample for AU Core FHIR package usage

### PR DESCRIPTION
**Pull request by sameeragunarathne**

### CODE REVIEW

File: working-with-fhir/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+```bash
+$ bal run
+```
+
+## Usage
+
+The project contains samples using Ballerina FHIR packages. Currently following two samples are available in this project:
+
+1. FHIR payload parsing: Parsing a FHIR resoource payload to respective ballerina FHIR record using Ballerina FHIR parser.
```
Comment (Thiyanwso2) [inline working-with-fhir/README.md:26]: Type @ resoource

File: working-with-fhir/main.bal [hunk 2]
```diff
@@ -0,0 +1,6 @@
+public function main() returns error? {
+    // Show case how to parse FHIR payloads
```
Comment (Thiyanwso2) [inline working-with-fhir/main.bal:2]: Type **Show case** -> Showcase

File: working-with-fhir/main.bal [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    check fhirDataParsing();
+    // Show case how to use datamapper to map custom data to FHIR payloads
```
Comment (Thiyanwso2) [inline working-with-fhir/main.bal:4]: Type **Show case** -> Showcase

**Review by Thiyanwso2 — CHANGES_REQUESTED** [review]

File: working-with-fhir/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# FHIR Data Mapping Sample
+
+This repository contains a sample implementation of FHIR data mapping using Ballerina data mapper.
+
+## Prerequisites
+
+Before running this sample, make sure you have the following:
+
+- Download and install [Ballerina Swan Lake](https://ballerina.io/downloads/) 2202.8.5 or above.
```
Comment (isuruh15) [inline working-with-fhir/README.md:9]: ballerina version should be 2201.8.5

File: working-with-fhir-igs/au-core/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Before running this sample, make sure you have the following:
+
+- Download and install [Ballerina Swan Lake](https://ballerina.io/downloads/) 2201.8.5 or above.
+
+## Getting Started
+
+To get started with this sample, follow these steps:
+
+1. Clone this repository to your local machine.
+2. Run the project using the following command:
```
Comment (isuruh15) [inline working-with-fhir-igs/au-core/README.md:16]: Do we need a step asking to navigate to the relevant project?

Comment (sameeragunarathne) [reply working-with-fhir-igs/au-core/README.md]: +1. added change in f68c3145f92c07b83a485f31c7d0b451dc782aba

**Review by isuruh15 — APPROVED** [review]

**Review by Thiyanwso2 — APPROVED** [review]

**Pull request merged**